.werft/util/kubectl.ts

```diff
@@ -1,23 +1,32 @@
 import { ShellString } from 'shelljs';
-import { exec, ExecOptions } from './shell';
+import { exec, ExecOptions, werft } from './shell';
 
 
+export const IS_PREVIEW_APP_LABEL: string = "isPreviewApp";
+
 export function setKubectlContextNamespace(namespace, shellOpts) {
     [
         "kubectl config current-context",
         `kubectl config set-context --current --namespace=${namespace}`
     ].forEach(cmd => exec(cmd, shellOpts));
 }
 
-export function wipeAndRecreateNamespace(helmInstallName: string, namespace: string, shellOpts: ExecOptions) {
+export async function wipeAndRecreateNamespace(helmInstallName: string, namespace: string, shellOpts: ExecOptions) {
```

Comment:
This is left here mainly for backwards compatibility for now (`com` uses this, too!)